xcat-inventory/xcclient/inventory/invdiff.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+from __future__ import print_function
+import deepdiff
+import pprint
+import yaml
+import json
+import sys
+
+
+def loadfile(filename):
+    contents={}
+    with open(filename,"r") as fh:
+        f=fh.read()
+        try:
+            contents=json.loads(f)
+        except ValueError:
+            try:
+                contents = yaml.load(f)
+            except Exception,e:
+                raise InvalidFileException("Error: failed to load file \"%s\", please validate the file with 'yamllint %s'(for yaml format) or 'cat %s|python -mjson.tool'(for json format)!"%(location,location,location))
+        return contents
+    return None
+
+def dump2yaml(xcatobj, location=None):
```

Comment:
What is the purpose of this script? Will be part of xcat-inventory?
If yes, the dump2yaml and dump2json have been defined in manager.py, can be reused.